src/barkhaPJun20/ArrayListFunctions_RemoveWordFromArrayList.java

```diff
@@ -0,0 +1,40 @@
+/* 1 : 
+Remove all Techno from given ArrayList.
+["Maulik","Techno" ,"Krishna", "Techno", "Credits", "Techno"]
+Hint : contains & remove*/
+
+package barkhaPJun20;
+
+import java.util.ArrayList;
+
+public class ArrayListFunctions_RemoveWordFromArrayList {
+	//method 1
+	void removeWordfromArrayList(ArrayList <String>arrayList, String removeWord) {
```

Comment:
You can manage whole logic in single for loop, like below

you have iterate on for loop and check in if condition 

if (names.get(index).equals(indexName)) {
	names.set(names.indexOf("Techno"), "Credits");
	countReplacement++;
} else if (names.get(index).equals(replacement)) {
        countReplacement++;
}

Reply:
Incorporated changes as suggested